server/handler/pull_request.go

```diff
@@ -69,14 +69,18 @@ func (h *PullRequest) Handle(ctx context.Context, eventType, deliveryID string,
 	}
 
 	if event.GetAction() == "labeled" || event.GetAction() == "opened" {
-		base, _ := pullCtx.Branches()
-		didUpdatePR, err := h.UpdatePullRequest(logger.WithContext(ctx), pullCtx, client, config, pr, base)
-		if err != nil {
-			logger.Error().Err(errors.WithStack(err)).Msg("Error updating pull request")
-		}
-
-		if didUpdatePR {
-			return nil
+		if !h.DisableUpdateFeature {
```

Comment:
Minor style thing: I might invert this condition so that the skip/log happens in the `if` and the update happens in the `else`.